lib/rspec/core/project_initializer/spec/spec_helper.rb

```diff
@@ -59,7 +59,7 @@
   # recommended. For more details, see:
   #   - http://myronmars.to/n/dev-blog/2012/06/rspecs-new-expectation-syntax
   #   - http://www.teaisaweso.me/blog/2013/05/27/rspecs-new-message-expectation-syntax/
-  #   - http://myronmars.to/n/dev-blog/2014/05/notable-changes-in-rspec-3#new__config_option_to_disable_rspeccore_monkey_patching
+  #   - http://myronmars.to/n/dev-blog/2014/05/notable-changes-in-rspec-3#zero-monkey-patching-mode
```

Comment:
Actually, that redirects to rspec.info.  Mind changing it to http://rspec.info/blog/2014/05/notable-changes-in-rspec-3/#zero-monkey-patching-mode ?

While you're at it, the link 2 lines above it redirects to http://rspec.info/blog/2012/06/rspecs-new-expectation-syntax/ if you want to update that as well.